deployments/ansible/roles/splunk-otel-collector/README.md

```diff
@@ -0,0 +1,119 @@
+- `collector_config_source`: Source path to a Splunk OTel Collector config YAML 
+  file on a control node that will be uploaded and set in place of
+  `collector_config` in remote nodes. Can be used to submit a custom collector 
+  config, e.g. `./custom_collector_config.yaml`. (**default:** `""` meaning 
+  that nothing will be copied and existing `collector_config` will be used)
+
+- `splunk_bundle_dir`: The path to the [Smart Agent bundle directory](
+  https://github.com/signalfx/splunk-otel-collector/blob/main/internal/extension/smartagentextension/README.md).
+  The default path is provided by the collector package. If the specified path
+  is changed from the default value, the path should be an existing directory
+  on the node. The `SPLUNK_BUNDLE_DIR` environment variable will be set to
+  this value for the collector service.  (**default:**
+  `/usr/lib/splunk-otel-collector/agent-bundle`)
+
+- `splunk_collectd_dir`: The path to the collectd config directory for the
+  Smart Agent bundle. The default path is provided by the collector package.
+  If the specified path is changed from the default value, the path should be
+  an existing directory on the node. The `SPLUNK_COLLECTD_DIR` environment
+  variable will be set to this value for the collector service.  (**default:**
+  `/usr/lib/splunk-otel-collector/agent-bundle`)
+
+- `service_user` and `service_group` (Linux only): Set the user/group
+  ownership for the collector service. The user/group will be created if they
+  do not exist. (**default:** `splunk-otel-collector`)
+
+- `memory_total_mib`: Amount of memory in MiB allocated to the Splunk OTel 
+  Collector. (**default:** `512`)
+
+- `ballast_size_mib`: Memory ballast size in MiB that will be set to the Splunk 
+  OTel Collector. (**default:** 1/3 of `memory_total_mib`)
+
+- `install_fluentd`: Whether to install/manage fluentd and dependencies for log
+  collection. The dependencies include [capng_c](
+  https://github.com/fluent-plugins-nursery/capng_c) for enabling
+  [Linux capabilities](
+  https://docs.fluentd.org/deployment/linux-capability),
+  [fluent-plugin-systemd](
+  https://github.com/fluent-plugin-systemd/fluent-plugin-systemd) for systemd
+  journal log collection, and the required libraries/development tools.
+  (**default:** `true`)
+
+- `td_agent_version`: Version of td-agent (fluentd package) that will be 
+  installed (**default:** `3.3.0` for Debian jessie, `3.7.1` for Debian 
+  stretch, and `4.1.1` for other distros`)
+
+- `fluentd_config`: Path to the fluentd config file on the remote host.
+  (**default:** `/etc/otel/collector/fluentd/fluent.conf`)
+
+- `fluentd_config_source`: Source path to a fluentd config file on a 
+  control node that will be uploaded and set in place of `fluentd_config` on
+  remote nodes. Can be used to submit a custom fluentd config,
+  e.g. `./custom_fluentd_config.conf`. (**default:** `""` meaning 
+  that nothing will be copied and existing `fluentd_config` will be used)
+
```

Comment:
Should we include `environment` option?

Reply:
It's started on parity with existing linux deployment. we can add environment later, it'll require switching from reusing the same config to a template that has to be kept in sync with regular deployment - so need to decide if we want that